cinnabar/tests/test_from_experiment.py

```diff
@@ -0,0 +1,14 @@
+from openff.units import unit
+import pytest
+from cinnabar import measurements
+
+def test_Ki_to_DG():
```

Comment:
Isn't this already under `test_measurements`?

Reply:
Yes, that was a mistake, removed this.